### PR TITLE
help finalizing home page

### PR DESCRIPTION
**Pull request by bautistaaa**
changes
- auto change the challenge creation slides
- tweak ff providers and adds to env example
- update contributors

https://github.com/bautistaaa/typehero/assets/3660667/2594c564-646d-4c58-89d3-7f52c42629d6

### CODE REVIEW

**Comment by bautistaaa** [comment]
hmm i dont like it anymore im closing

**Comment by mirashif** [comment]
sure, we can auto change the challenge creation card.

but, what did you mean by light flicker?

**Comment by mirashif** [comment]
oh, i see the problem now lol. we'll fix it today.

**Comment by PickleNik** [comment]
yop, kill the light flicker i agree

**Comment by PickleNik** [comment]
let that amogus loop tho 🥸

**Comment by PickleNik** [comment]
oh nvm i see now it stays up there

**Review by PickleNik — APPROVED** [review]
LGTM

**Comment by PickleNik** [comment]
actually is it possible to have both the click to skip step and the auto-skipping be a thing?

**Comment by PickleNik** [comment]
for card 3

File: apps/web/src/components/landing/feature-card.tsx [hunk 1]
```diff
@@ -354,11 +354,6 @@ export function ChallengeCreationCard({
           />
         </>
       )}
-      {/* whole card is a "Next button" */}
-      <div
```
Comment (PickleNik) [inline apps/web/src/components/landing/feature-card.tsx:358]: I'd keep dis cuz it's engaging to keep clicking

**Comment by PickleNik** [comment]
image

**Pull request merged**